core-web/apps/dotcms-ui/src/app/view/components/login/reset-password-component/reset-password.component.html

```diff
@@ -25,10 +26,14 @@ <h3 data-testId="header">{{ loginInfo.i18nMessagesMap['reset-password'] }}</h3>
             </dot-field-validation-message>
         </div>
         <div class="field form__group--validation">
-            <label data-testId="confirmLabel">{{
-                loginInfo.i18nMessagesMap['re-enter-password']
-                }}</label>
+            <label
+                class="p-label-input-required"
```

Comment:
If we have the `directive` now, should we use it here? To keep consistency.

Reply:
We used directive only for dynamic fields because we want to keep it simple other wise we need to change whole application.